layout-webapp/src/main/webapp/groovy/portal/webui/container/UIPageLayout.gtmpl

```diff
@@ -17,8 +17,12 @@
     for (int i =0; i < portlets.size(); i++) {
       def portletId = portlets.get(i).getId();
       if (portletId != null) {
+        def url=rcontext.getRequest().getRequestURI()+"?maximizedPortletId="+portletId+"&showMaxWindow=true&hideSharedLayout=true&maximizedPortletMode=VIEW";
+        for (String parameter : rcontext.getRequest().getParameterMap().keySet()) {
+          url += "&" + parameter + "=" + rcontext.getRequest().getParameter(parameter);
+        }
```

Comment:
It was made on purpose to not propagate Request Params. Let's discuss about options if needed.

```suggestion
```